lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -220,8 +220,9 @@ def authorize(money, payment_source, options = {})
       end
 
       def verify(creditcard, options = {})
+        amount = allow_0_auth?(creditcard, options) ? 0 : 1
```

Comment:
❓ Question:

Maybe I'm wrong but as far as I remember values are passed as cents, that's why the initial value was 100, not 1, perhaps the values for the ternary operation should be 0 and 100 ?

Reply:
Thx, you're righ, fixed